scripts/revision-rapida.py

```
@@ -0,0 +1,628 @@
+                archivo=str(ruta.relative_to(RAIZ)),
+                linea=linea,
+                severidad="error",
+                categoria="Formato LaTeX",
+                mensaje=f"Comando prohibido: `{m.group().strip()}`",
+                sugerencia=sugerencia,
+            ))
+    return problemas
+
+
+def analizar_etiquetas(ruta: Path, texto: str) -> list:
+    """Detecta figuras, tablas y ecuaciones sin \label{}."""
```

Ruff (0.15.9)

[warning] Invalid escape sequence: `\l`

Use a raw string literal

(W605)